listenbrainz/webserver/static/js/src/stats/UserEntityChart.tsx

```diff
@@ -553,9 +565,12 @@ export default class UserEntityChart extends React.Component<
                       }`}
                     >
                       <a
```

Comment:
These anchor tags aren't applied the disabled style correctly anymore and remain clickable, even though the `disabled` class is applied to the parent <li> element.
Not sure what's going on there…

Reply:
strange, I didn't notice this when I made the changes. I'll have a look again. Is it a bit weird that the css class is related to `disabled` being on the `<li>`? Maybe it should be on the anchor anyway, or perhaps we should update the logic to use a `<span>` instead of anchor when it's disabled

Reply:
This is because now that the anchor tag has an href, a rule for `a:visited` is overriding the rule that we have for this colour: https://github.com/metabrainz/listenbrainz-server/blob/f0bc6969ad8f5ddef1fb0d788998f2fcc45f3dad/listenbrainz/webserver/static/css/theme/links.less#L36

I was able to restore the correct colour by disabling these rules in the inspector:
![Screenshot from 2021-09-13 14-25-43](https://user-images.githubusercontent.com/19217/133083505-2a3ddfe9-a215-4d7e-8eb3-024f3ccbf0c1.png)

I don't know how to update the styles to make this work